cmake/Modules/drishti_set_unit_test_assets.cmake

```diff
@@ -1,52 +0,0 @@
-function(drishti_set_unit_test_assets
-    DRISHTI_ACF_FACE_MODEL
-    DRISHTI_FACE_LANDMARKER
-    DRISHTI_EYE_MODEL
-    DRISHTI_MEAN_FACE_LANDMARKS
-    )
-
-  set(ext "")
-  if(DRISHTI_SERIALIZE_WITH_CEREAL)
```

Comment:
Should all mentions of `DRISHTI_SERIALIZE_WITH_CEREAL`, `DRISHTI_SERIALIZE_WITH_BOOST`, `DRISHTI_USE_TEXT_ARCHIVES`, `DRISHTI_SERIALIZE_WITH_CVMATIO` be removed now?